### DISCUSSION TITLE
blog/getting-started-robot-sw-eng/

### DISCUSSION THREAD

**Discussion by giscus** [mikelikesrobots/mikelikesrobots.github.io#32] [Comments]
# blog/getting-started-robot-sw-eng/

This post is all about my advice on getting started as a Robotics Software Engineer. I want to tell you a little of my journey to get to this point, then what you should do to practise and give yourself the best start possible.

https://mikelikesrobots.github.io/blog/getting-started-robot-sw-eng/

**Comment by c0d33ngr** [comment]
Thank you for the article. I learnt a lot from it.

I'm taking some courses related to robotics and embedded systems from Coursera and it has been great so far coupled with the fact I got a pretty good programming skills. Though one of the courses teach with Webots open source simulatior but hasn't reach there yet.

**Reply by mikelikesrobots** [reply]
I'm really glad it was helpful. Let me know how helpful the Webots is - I could look at that for one of my posts as well.
